bundles/pixi.js-node/README.md

```diff
@@ -88,6 +99,31 @@ const output = `./test.png`;
 writeFileSync(output, base64Data, 'base64');
 ```
 
+## Full environment setup with Docker 🐳
+```Dockerfile
+# Set the base image
+FROM node:16
+
+# Create and set the working directory
+WORKDIR /usr/src/app
+
+# Add dependencies for gl, canvas and xvfb
+# Important! These dependencies must be installed before running `npm install`
+RUN apt-get update
+RUN apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev xvfb
```

Comment:
```suggestion
RUN apt-get update \
    && apt-get install -y build-essential libcairo2-dev libgif-dev libglew-dev libglu1-mesa-dev libjpeg-dev libpango1.0-dev librsvg2-dev libxi-dev pkg-config xvfb \
    && rm -rf /var/lib/apt/lists/*
```

This is recommended in [Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get).